fusor-ember-cli/app/routes/subscriptions/select-subscriptions.js

```diff
@@ -56,44 +78,18 @@ export default Ember.Route.extend({
 
   actions: {
 
-    saveSubscriptions(redirectPath) {
```

Comment:
this was the previous action `saveSubscriptions` that deleted rows in Fusor::Subscription and then re-created them, which caused problems.